src/renderer/src/components/Messages/VoiceInput.tsx

```
@@ -17,13 +53,336 @@ export default function VoiceInput({
+        content: text,
+        evaluate: true,
+        sender: participantId,
+        audios,
+      }),
+    );
+    audioChunks.map((chunk, index) => deleteAudioChunk(index));
+    setAudioChunks([]);
+  };
+
+  const handleStartRecording = async () => {
+    if (navigator.mediaDevices && navigator.mediaDevices.getUserMedia) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)